Schema.js

```diff
@@ -75,6 +78,7 @@ function classNameIsValid(className) {
     className === '_Session' ||
     className === '_SCHEMA' || //TODO: remove this, as _SCHEMA is not a valid class name for storing Parse Objects.
     className === '_Role' ||
+    className === '_GlobalConfig' ||
```

Comment:
Looks like you are right. Maybe we need a `database.rawCollection()` function or something like that, to bypass the `classNameIsValid` check.